index.rst

```diff
@@ -1,58 +1,163 @@
+
+
+Ongoing support
+---------------
+
+Support is offered to observatory staff on the #com-square Slack channel.
+
+Work is co-ordinated through the appropriate meeting (currently the Comissioning Activities Project or CAP meeting) and carried out according the process outlined in https://sqr-056.lsst.io
+
+
+Handover
+========
+
+Delivering team
+----------------
+
+SQuaRE, with contributions from the SUI/IPAC team.
+
+Receiving Operations team
+--------------------------
+
+SQuaRE (as the SUI/IPAC effort is part of SQuaRE in the Ops WBS)
+
+Authorization or asset transfer
+-------------------------------
+
+None required.
+
+Known Issues
+============
+
+There are no current issues affecting the quality of the service.
+
+There are occasionally tickets asking for improvements or a better user experience that are specific to the telescope usage of the service (see `[DM-30151] <https://jira.lsstcorp.org/browse/DM-30151>`_ for an example) and recognizing the importance of the service in summit operations we make a concerted attempt to prioritise them in Construction and will continue to do so in Operations.
+
+Additional Notes
+================
+
+Authentication
+--------------
+
+There are two currently implemented ways of authenticating (establishing user identity) to the Science Platform, as supported by the A&A service (gafaelfawr).
+
+These are:
+
+- CILogon, in this case tied to the NCSA identity LDAP
+- Github tied to specific teams, in this case `rubin-summit/teams/rsp-access <https://github.com/orgs/rubin-summit/teams/rsp-access>`_ among others.
+
+Our summit IT team is planning on providing us with an OAuth2 service backed the FreeIPA identity service.
+When this is available, work will be needed to interface to that service, which will allow us to operate the summit instance in the absence of external network.
+This work can be better prioritised as part of Operations, since it is not part of Construction delivery.
+
+Telescope & Site Build/Release
+-------------------------------
+
+This service consumes containers built by Telescope & Site (in Operations, Observatory Operations) and SQuaRE collaborates with the Telescope & Site release co-ordinator and scientist on adding the JupyterLab layer on top of those containers for the notebook service (nublado). We occasionally meet to resolve issues across (or about) our interface, eg our recent `Build Workshop <https://confluence.lsstcorp.org/display/DM/2021-04-14+Build+Workshop>`_
+
+Observatory IT
+--------------
+
+This service consumes containers (specifically multus init containers) provided through Observatory IT for supporting multicast networking for our notebook pods.
+
+Like all of SQuaRE's summit services, RSP at the summit and base relies completely on Kubernetes infrastructure and network supported by Observatory (Chile) IT.
+
+Visual Identity
+---------------
+
+The theming elements of Square One, the umbrella application for RSP users, contains visual identity elements.
```

Comment:
When this becomes Operations, does the VI coincide with Ops?

Reply:
(sorry it's taken so long to get back to this) Yes, the visual identity dictionary is just a technical representation of the visual identity assets that were made available to us by Communications. We will update them if they change. I believe (I can check) that SQuaRE One is already "skinned" for Ops, eg. with the Ops agency acknowledgements. 

Oh, it occurs to me that I misunderstood the question. If the question is whether our visual identity dictionary is part of this RTN delivery then no, it is a different asset, we will deliver than with our documentation system.